.github/workflows/test.yml

```
@@ -58,14 +76,24 @@ jobs:
           SKIP: local-test-core-prep,local-test-pip-prep,local-testing,local-quality
 
   # Prepare default python version environment
-  ha-core-prepare:
+  ha-core-release:
     runs-on: ubuntu-latest
-    name: Setup for HA-core (release)
+    name: Setup for HA-core (release/master)
     needs:
       - pre-commit
+    outputs:
+      release_failed: ${{ steps.ha_core_tests.outputs.failed }}
```

actionlint (1.7.4)

property "ha_core_tests" is not defined in object type {cache-hacore: {conclusion: string; outcome: string; outputs: {string => string}}; determine_mode: {conclusion: string; outcome: string; outputs: {string => string}}; ha-core-tests: {conclusion: string; outcome: string; outputs: {string => string}}; python: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)